src/index.js

```diff
@@ -0,0 +1,62 @@
+import './modules/style.css';
+import './modules/linkedList-module.js';
+
+const array = document.querySelector('.list');
+const { LinkedList } = require('./modules/linkedList-module.js');
+
+const list = new LinkedList();
+
+const arr = [
+  {
+    value: 'Todo1',
+    completed: false,
+    index: 0,
+  },
+  {
+    value: 'Todo2',
+    completed: false,
+    index: 1,
+  },
+  {
+    value: 'Todo3',
+    completed: false,
+    index: 2,
+  },
+  {
+    value: 'Todo4',
+    completed: false,
+    index: 3,
+  },
+  {
+    value: 'Todo5',
+    completed: false,
+    index: 4,
+  },
+];
```

Comment:
- [ ] Kindly Make sure these todos are visible in the browser.

![image](https://user-images.githubusercontent.com/65068771/177629948-1fc78eca-185e-4792-910c-ce72389c9328.png)